IBAutomater updates

StefanoRaggi opened the pull request:
- Improve detection of authenticating/connecting state
- Log contents of unknown windows also if there are no labels or text panes

StefanoRaggi committed:
Improve detection of authenticating/connecting state

Martin-Molinero commented on QuantConnect.IBAutomater/IBAutomater.cs:
```diff
@@ -518,10 +518,12 @@ private void OnProcessOutputDataReceived(string text)
-                    _isAuthenticating = text.Contains("Authenticating");
+                    _isAuthenticating = text.Contains("Authenticating") ||
+                                        text.Contains("Connecting to server") ||
+                                        text.Contains("server error, will retry");
```
nit - think these types of check should be case insensitive?

Martin-Molinero reviewed (commented):
Mind expanding which error cases required this change?

StefanoRaggi commented:
> Mind expanding which error cases required this change?

When at startup IBGateway tries to connect to the server, the title of the splash window can change before asking for credentials,
these are some of the different strings I've seen in separate log files (although they all indicate that IBGateway is trying to connect to the server):
```
Window event: [WINDOW_OPENED] - Window title: [Authenticating (trying for another 19 seconds)...] - Window name: [frame2]
Window event: [WINDOW_OPENED] - Window title: [Attempt 1: server error, will retry in seconds...] - Window name: [frame2]
Window event: [WINDOW_OPENED] - Window title: [Connecting to server...] - Window name: [frame2]
Window event: [WINDOW_OPENED] - Window title: [Connecting to server (trying for another 19 seconds)...] - Window name: [frame2]
```

The only reason for handling these messages is to get a better error message if not able to connect:
```
The IBAutomater initialization timed out. 
The logon process could not be completed because the IB server is busy or being reset for the weekend, 
please try again later.
```
instead of this one:
```
The IBAutomater initialization timed out.
```

StefanoRaggi committed:
Update IBAutomater to v2.0.62

StefanoRaggi committed:
Update to IBAutomater v2.0.63

- case-insensitive string comparisons
- upgrade target framework from netstandard20 to netstandard21
